javascript-timers/interval.js

```diff
@@ -0,0 +1,17 @@
+var textEl = document.querySelector('.countdown-display');
+
+function changeCount() {
+  if (textEl.textContent === '4') {
+    textEl.textContent = '3';
+  } else if (textEl.textContent === '3') {
+    textEl.textContent = '2';
+  } else if (textEl.textContent === '2') {
+    textEl.textContent = '1';
+  } else if (textEl.textContent === '1') {
```

Comment:
What if the timer started from 1000? You should try to find a way to do this that doesn't require a conditional for each number.